products/qt.md

```diff
@@ -30,7 +30,6 @@ releases:
     extendedSupport: 2026-03-31
     latest: "6.5.3"
     latestReleaseDate: 2023-09-28
-    link: https://www.qt.io/blog/qt-6.5-lts-released
```

Comment:
I think this not a commercial version.

Reply:
What do you mean ? I dropped the link because https://www.qt.io/blog/qt-6.5.3-released can be generated by the `changelogTemplate`.

Reply:
Sorry for the confusion, I was not aware of the template :smile: -> resolved